lib/LoginStrategy.ts

```diff
@@ -75,6 +82,13 @@ export class LoginStrategy extends BaseStrategy {
       acceptedRoles,
       ...options
     }: LoginStrategyOptions,
+    private issuerResolver: IssuerResolver,
+    private revokerResolver: RevokerResolver,
+    private credentialResolver: CredentialResolver,
+    private verifyProof: (
```

Comment:
It seems like we don't need to hold on to references to these classes beyond the constructor
```suggestion
    issuerResolver: IssuerResolver,
    revokerResolver: RevokerResolver,
    private credentialResolver: CredentialResolver,
    verifyProof: (
```